src/main/java/org/opensearch/security/configuration/ConfigurationRepository.java

```diff
@@ -242,18 +242,18 @@ private void waitForSecurityIndexToBeAtLeastYellow() {
             try {
                 response = client.admin().cluster().health(new ClusterHealthRequest(securityIndex).waitForYellowStatus()).actionGet();
             } catch (Exception e) {
-                LOGGER.debug("Caught again a {} but we just try again ...", e.toString());
+                LOGGER.info("Caught again a {} but we just try again ...", e.toString());
             }
         }
     }
 
     public void initOnNodeStart() {
         try {
-            if (settings.getAsBoolean(ConfigConstants.SECURITY_ALLOW_DEFAULT_INIT_SECURITYINDEX, false)) {
+            if (settings.getAsBoolean(ConfigConstants.SECURITY_ALLOW_DEFAULT_INIT_SECURITYINDEX, true)) {
```

Comment:
Instead of changing the default behavior can we instead update the test setup in the GHA?

Why would only Windows be affected whereas Linux doesn't seem to have issues?